lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -334,6 +334,21 @@ def gem_version_compare(first, second)
             Gem::Version.new(first.to_s) <=> Gem::Version.new(second.to_s)
           end
 
+          apipie :method, 'return a string that can be sorted' do
+            required :version, String, desc: 'package version'
+            returns String, desc: 'string that can be sorted'
+            example 'sortable_version("1.5.2-1") #=> 01-1.01-5.01-2.01-1'
+            example 'sortable_version("") #=> ""'
+          end
```

Comment:
Just a nitpick: can you please capitalize the descriptions?

Reply:
@ofedoren, capitalize all the desc and update the PR.